core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -33,6 +33,10 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
     apiClientProvider.getWorkspaceApplicationApi(accessToken.token)
   }
 
+  private def getControlledAzureResourceApi(accessToken: OAuth2BearerToken) = {
```

Comment:
There is an [HttpWorkspaceManagerDAOSpec ](https://github.com/broadinstitute/rawls/blob/develop/core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAOSpec.scala) that we can use to get coverage on this method and the other new ones in this class.

Reply:
Added coverage in f3b873109639ff47b6e246d39d8d2fdde696b185.